projects/exui-common-lib/src/lib/components/search-location/search-location.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="govuk-form-group">
+    <div class="location-picker-custom">
+        <label class="govuk-body" for="location-picker">
+            Search for a location by name
+        </label>
+        <br>
+        <div style="width: 50%; display: inline-block; margin-right: 4px;">
+            <div>
+                <div class="autocomplete__wrapper">
+                    <div style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; margin-bottom: -1px; margin-right: -1px; overflow: hidden; padding: 0px; position: absolute; white-space: nowrap; width: 1px;"><div id="location-picker__status--A" role="status" aria-atomic="true" aria-live="polite"></div><div id="location-picker__status--B" role="status" aria-atomic="true" aria-live="polite"></div></div>
+                        <input 
+                            [formControl]="findLocationForm.controls.findLocationControl"
+                            [matAutocomplete]="auto"
+                            aria-expanded="false" 
+                            aria-owns="location-picker__listbox" 
+                            aria-autocomplete="both" 
+                            aria-describedby="location-picker__assistiveHint" 
+                            autocomplete="off" 
+                            class="autocomplete__input autocomplete__input--default search-box"
+                            [attr.disabled]="disabled"
+                            [value]="selectedLocation?.court_name"
+                            id="input-selected-location">
+                        <mat-autocomplete class="mat-autocomplete-panel-extend" autoActiveFirstOption #auto="matAutocomplete">
+                            <mat-option *ngFor="let location of locations$ | async"
+                                        [value]="getDisplayName(location)"
+                                        (onSelectionChange)="onSelectionChange(location)"
+                                        [ngClass]="{'hide-autocomplete': !showAutocomplete}">
+                                {{ getDisplayName(location) }}
+                            </mat-option>
+                        </mat-autocomplete>
+                </div>
+            </div>
+        </div>
+        <a href="javascript:void(0)" (click)="addSelection()" class="govuk-button govuk-button--secondary" data-module="govuk-button">
+            Add location
+        </a>
+    </div>
+    <ul  *ngIf="selectedLocations$ && (selectedLocations$ | async).length" class="hmcts-filter-tags selection-container">
```

Comment:
It can be simplified to:
(selectedLocations$ | async)?.length
instead of **&&** in a pipe.